diff --git a/app/layout.tsx b/app/layout.tsx
index 2529490..1d44d36 100644
--- a/app/layout.tsx
+++ b/app/layout.tsx
@@ -19,6 +19,7 @@ export const metadata: Metadata = {
   description: 'Authorized members only.',
   robots: { index: false, follow: false },
   icons: { icon: '/logo.png', shortcut: '/logo.png', apple: '/logo.png' },
+  verification: { google: '7VfuwzozReuUwaodd5kmJAMF9HbifQWHX0aU-EUfpYg' },
 };
 
 export default function RootLayout({ children }: { children: React.ReactNode }) {
